ci/travis/build_project.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
I am not sure, does this commit belong in this PR?

Reply:
this was my mistake trying to solve a conflict, fixed now